include/small/matras.h

```diff
@@ -289,6 +303,13 @@ matras_alloc_range(struct matras *m, matras_id_t *id, matras_id_t range_count);
 void
 matras_dealloc_range(struct matras *m, matras_id_t range_count);
 
+/**
+ * Reserve the max amount of extents required to successfully touch @p count
+ * blocks. The extents are reserved in the allocator given on construction.
+ */
+int
+matras_touch_reserve(struct matras *m, unsigned count);
```

Comment:
Let's make it back `int` then for consistensy.  Sorry, it was `int` before my first review AFAIR but back then I was not know our conventions for types for count and similar.

Reply:
Done.